src/fabric.c

```diff
@@ -446,6 +446,7 @@ void fi_ini(void)
 
 	ofi_register_provider(UDP_INIT, NULL);
 	ofi_register_provider(SOCKETS_INIT, NULL);
+	ofi_register_provider(TCP_INIT, NULL);
```

Comment:
We should move the tcp provider before sockets in the ordered list.  By the time 1.7 is released, it just needs to be as stable and better performing.